project_buttom_kanban_open_form_view/views/project_project.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="project_buttom_kanban_view" model="ir.ui.view">
+        <field name="name">project.project.kanban</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project_kanban" />
+        <field name="arch" type="xml">
+            <xpath expr="//span[@class='o_text_overflow']" position="after">
+                <a name="action_project_project_view_form" type="object">
+                    <span class="o_text_overflow"><strong><t t-esc="record.name.value"/></strong></span>
+                </a>
+            </xpath>
+            <xpath expr="//span[@class='o_text_overflow']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
```

Comment:
Try this (safer, but not tested), because there are more than one `span` whose class is `o_text_overflow` within this view:
```suggestion
            <xpath expr="//div/(span[@class='o_text_overflow'])[1]" position="after">
                <a name="action_project_project_view_form" type="object">
                    <span class="o_text_overflow"><strong><t t-esc="record.name.value"/></strong></span>
                </a>
            </xpath>
            <xpath expr="//div/(span[@class='o_text_overflow'])[1]" position="attributes">
                <attribute name="invisible">1</attribute>
            </xpath>
```